lib/src/element_type.dart

```diff
@@ -23,7 +23,11 @@ class ElementType {
 
   bool get isParameterizedType {
     if (_type is FunctionType) {
-      return (_type as FunctionType).typeFormals.isNotEmpty;
+      final FunctionType functionType = _type;
+      final Iterable<DartType> typeArguments = []
+          ..addAll(functionType.typeArguments)
+          ..addAll(functionType.typeFormals.map((f) => f.type));
+      return typeArguments.isNotEmpty;
     } else if (_type is ParameterizedType) {
```

Comment:
This getter (and the one below) looks incorrect to me, but I'm not 100% sure how this getter will be used.

It will break once we have typedefs for generic function types, and it will break for generic function typed parameters (e.g. `foo(T genericFn<T>(T x))`). Mixing typeArguments and typeFormals is never a good idea.

If the goal is to display instantiated typedefs using their `Name<PlusGenericArguments>` then it should be looking at the FunctionType's element and handling the FunctionTypeAliasElement case (for typedefs) and the FunctionTypedFormalParameter (for function typed formals).